paymentsheet-example/src/main/res/values/strings.xml

```diff
@@ -2,6 +2,7 @@
     <string name="app_name">PaymentSheet Example</string>
     <string name="paymentsheet">PaymentSheet</string>
     <string name="paymentsheet_custom">PaymentSheet Custom</string>
+    <string name="paymentsheet_serverside_confirmation">PaymentSheet with server-side confirmation</string>
```

Comment:
We're quite inconsistent with our localizations in our examples. Do we care?

Reply:
We probably should, at least a little. I’ll clean this up.